### PR TITLE
Travis setup

### PR DESCRIPTION
**Pull request by javagl**
This is supposed to set up travis. (This includes adding a `coverage` functionality with `nyc`. These could be independent PRs, but testing the coverage should be part of CI, so I hope it makes sense to _cover_ this here as well)

Trying this out locally would apparently require setting up a certain toolchain (e.g. as in https://stackoverflow.com/a/49019950 - with brittle, tedious, manual steps, copy-and-paste of job IDs and whatnot...) that makes me think "Well, just let it run on the server and see whether it works..." - but I'm open for either approach here...

If this "works", it fixes https://github.com/CesiumGS/3d-tiles-validator/issues/7 ...

### CODE REVIEW

**Comment by javagl** [comment]
It's building. It's passing. I'll just mark this as "Ready For Review".

(An aside: The coverage does not look so great right now. This is to a large extent caused by the `tileFormats/(legacy)` part, and by some low-level helper classes that have been ported from CesiumJS and from which no all functions are actually used by the validator. Increasing the coverage e.g. by adding CMPT/PNTS/I3DM files in the `specs` tests, or removing the unused code paths could be considered as an ongoing task)

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /node_modules
 /build
 /dist
+/coverage
+/.nyc_output
```
Comment (lilleyse) [inline .gitignore:5]: These should also be added to `.npmignore`

**Comment by lilleyse** [comment]
For new repos I think we should be using GitHub Actions. The setup should be pretty minimal.

**Comment by javagl** [comment]
I converted travis into a GitHub action. This will only be run _after_ the corresponding file was merged (so it's even more difficult to test this beforehand), but from my first, basic experiments, the usage should be pretty straightforward (and if there's an issue with the YML, I'd fix that quickly)

**Comment by lilleyse** [comment]
[ubuntu-latest](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md) comes preinstalled with Node 16 so it should be ok, but we'll see

**Comment by lilleyse** [comment]
I pushed a couple commits directly to main to get it working: https://github.com/CesiumGS/3d-tiles-validator/commits/main

**Comment by javagl** [comment]
Seems to work 🎉

**Pull request merged**